.github/workflows/kernel_tests.yaml

```diff
@@ -34,8 +33,21 @@ jobs:
           name: ansible_output_${{matrix.architecture}}
           path: ansible_output.tar
           
+      - name: Build matrix_gen
+        working-directory: matrix_gen
+        run: go build .
+        
+      - name: Generate new matrix
+        working-directory: matrix_gen
+        run: ./matrix_gen
+        
+      - uses: actions/upload-artifact@v3
```

Comment:
For now, just upload the matrix.
We will need to deploy it as static file to gh pages, somewhat.